sdk/template-dpg/Azure.ServiceTemplate.Template/content/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why do we need this file?  Could we copy it in from somewhere else?

Reply:
Confirmed with Wes, the file is not needed. Removed it.